drivers/interrupt_controller/exti_stm32.c

```diff
@@ -270,7 +304,6 @@ static inline void __stm32_exti_isr_15_10(void *arg)
 	__stm32_exti_isr(10, 16, arg);
 }
 
-#ifdef CONFIG_SOC_SERIES_STM32F4X
```

Comment:
seems like this shouldn't be removed

Seems like #if defined(CONFIG_SOC_SERIES_STM32F0X) || defined(CONFIG_SOC_SERIES_STM32F4X)